test/inputs-ante-bytes-test.js

```diff
@@ -15,6 +15,55 @@ describe('ante-bytes', () => {
     expect(bytes.check({type: 'antebytespreview'})).to.be.true;
   });
 
+  it('recognizes and formats the antebytes record from dt-router', () => {
```

Comment:
Is this just to check that the new impression fields (placements_key, target_path, zone_name) get in there?